packages/shared/components/molecules/AssetIcon.svelte

```diff
@@ -86,7 +86,13 @@
     </div>
     <span class="absolute flex justify-center items-center bottom-0 right-0">
         {#if asset.verification.verified === true}
-            <NetworkIconBadge width={10} height={10} networkId={$activeProfile.network.id} {chainId} />
+            <NetworkIconBadge
+                width={10}
+                height={10}
+                networkId={$activeProfile.network.id}
+                {chainId}
+                tooltipText={$activeProfile?.network?.name}
```

Comment:
This should be changed to take into consideration the chain name if applicable.

![image](https://github.com/iotaledger/firefly/assets/45079109/eb862a43-deca-48a6-b760-1eb60d8209b9)